src/domains/wallet/hooks/use-import-options.ts

```diff
@@ -0,0 +1,75 @@
+import { NetworkManifestImportMethods } from "@arkecosystem/platform-sdk/dist/coins";
+import { useMemo } from "react";
+import { useTranslation } from "react-i18next";
+
+const convertMethodName = (methodName: string) => {
+	if (methodName === "bip38") {
+		return "encryptedWif";
+	}
+
+	if (methodName.startsWith("bip")) {
+		return "mnemonic";
+	}
+
+	return methodName;
+};
+
+interface Network {
+	options: string[];
+	defaultOption?: string;
+}
+
+export const useImportOptions = (methods: NetworkManifestImportMethods) => {
+	const { t } = useTranslation();
+
+	const allOptions = useMemo(
+		() => [
+			{ label: t("COMMON.MNEMONIC"), value: "mnemonic" },
```

Comment:
**Options**
```suggestion
			{ label: t("COMMON.MNEMONIC_BIP39"), value: "mnemonic.bip39" },
			{ label: t("COMMON.MNEMONIC_BIP44"), value: "mnemonic.bip44" },
			{ label: t("COMMON.MNEMONIC_BIP49"), value: "mnemonic.bip49" },
			{ label: t("COMMON.MNEMONIC_BIP84"), value: "mnemonic.bip84" },
```

**Labels**
```
COMMON.MNEMONIC_BIP39 => Mnemonic (Legacy)
COMMON.MNEMONIC_BIP44 => Mnemonic (Legacy)
COMMON.MNEMONIC_BIP49 => Mnemonic (SegWit)
COMMON.MNEMONIC_BIP84 => Mnemonic (Native SegWit)
```

BIP39 and BIP44 share the same labels because they generally aren't visible at the same time. There's a few exceptions but for those we will have a slightly different UX further down the road.

Reply:
this change breaks other things like:
https://github.com/ArkEcosystem/desktop-wallet/blob/develop/src/domains/wallet/pages/ImportWallet/Step2.tsx#L98-L112
https://github.com/ArkEcosystem/desktop-wallet/blob/develop/src/domains/wallet/hooks/use-wallet-import.ts#L27-L34

this needs new translate and structure
because we squash commits, i think it's better to be done on another PR, but i need this PR to be merged

Reply:
This is all related to the same underlying issue/change so do it in this PR.